README.md

```diff
@@ -13,6 +13,13 @@ V1.0 of the PCB layout:
 - Initialize libb6 submodule: `$ git submodule update --init`
 - Build Python extension for libb6: `$ make -C libraries`
 
+### Udev Setup
+Copy the .rules file (from the Documentation folder) into the folder on your computer which contains rules files for udev.
+
+For me it is: Computer/etc/udev/rules.d
```

Comment:
You can just say `/etc/udev/rules.d/`

Leading `/` signifies an absolute path

Reply:
Good to know!